src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -293,6 +293,48 @@ public void help() throws SpeechletException {
     }
 
     @Test
+    public void repeat(@Mocked ObaDao obaDao) throws SpeechletException, URISyntaxException, IOException {
```

Comment:
One question - if obaDao is mocked at the class level, should we still be passing it in here?